pkg/cmd/infra/router/router.go

```diff
@@ -37,35 +54,75 @@ type templateRouterConfig struct {
 	StatsUsername      string
 }
 
+// f5RouterConfig is the config necessary to start an F5 router plugin
+type f5RouterConfig struct {
+	Host     string
+	Username string
+	Password string
+	Vserver  string
+	Insecure bool
+}
+
+const (
+	//routerTypeF5 is the type value for an f5 router plugin
+	routerTypeF5 = "f5"
+	//routerTypeTemplate is the type value for a template router plugin
+	routerTypeTemplate = "template"
+)
+
+func newEmptyConfig() *routerConfig {
```

Comment:
Note that this is only necessary for exported vars, though it's good to have everything documented